packages/arb-validator/nodegraph/challengeSet.go

```diff
@@ -32,6 +33,59 @@ type Challenge struct {
 	conflictNode *structures.Node
 }
 
+func MakeChallenge(
+	_blockId *common.BlockId,
+	_logIndex uint,
+	_asserter common.Address,
+	_challenger common.Address,
+	_contract common.Address,
+	_conflictNode *structures.Node,
+) *Challenge {
```

Comment:
For go we don't use underscores for parameter names. Also, constructors should start with `New`. Maybe call this `NewChallenge` and then next one `NewChallengeFromEvent` if both are necessary.